tests/kafkatest/tests/streams/streams_application_upgrade_test.py

```diff
@@ -21,11 +21,12 @@
 from kafkatest.services.kafka import KafkaService
 from kafkatest.services.streams import StreamsSmokeTestDriverService, StreamsSmokeTestJobRunnerService
 from kafkatest.services.zookeeper import ZookeeperService
-from kafkatest.version import LATEST_2_2, LATEST_2_3, LATEST_2_4, LATEST_2_5, LATEST_2_6, LATEST_2_7, LATEST_2_8, DEV_VERSION, KafkaVersion
+from kafkatest.version import LATEST_2_2, LATEST_2_3, LATEST_2_4, LATEST_2_5, LATEST_2_6, LATEST_2_7, LATEST_2_8, \
+  LATEST_3_0, DEV_VERSION, KafkaVersion
 
 smoke_test_versions = [str(LATEST_2_2), str(LATEST_2_3), str(LATEST_2_4),
                        str(LATEST_2_5), str(LATEST_2_6), str(LATEST_2_7),
-                       str(LATEST_2_8)]
+                       str(LATEST_2_8), str(LATEST_3_0)]
```

Comment:
Shall we also add 
```
V_3_0_1 = KafkaVersion("3.0.1")
LATEST_3_0 = V_3_0_1
```
to `tests/kafkatest/version.py`?

Reply:
Good catch. Fixed.